client/packages/invoices/src/Prescriptions/api/api.ts

```diff
@@ -167,6 +189,37 @@ export const getPrescriptionQueries = (sdk: Sdk, storeId: string) => ({
 
     throw new Error('Could not delete invoices');
   },
+  updateLines: async (draftPrescriptionLine: DraftStockOutLine[]) => {
+    const input = {
+      insertPrescriptionLines: draftPrescriptionLine
+        .filter(
+          ({ type, isCreated, numberOfPacks }) =>
+            isCreated &&
+            type === InvoiceLineNodeType.StockOut &&
+            numberOfPacks > 0
+        )
+        .map(prescriptionParsers.toInsertLine),
+      updatePrescriptionLines: draftPrescriptionLine
+        .filter(
+          ({ type, isCreated, isUpdated }) =>
```

Comment:
Didn't add numberOfPacks check in case user only changes note (not sure if this will happen though?)

Reply:
worth keeping the check - currently you can cause 

<img width="275" alt="image" src="https://github.com/openmsupply/open-msupply/assets/9192912/41367af9-528a-4ac3-b16e-78ab2464d135">

just by adding a comment when there is no quantity